data/assets/actions/nightsky/misc.asset

```diff
@@ -0,0 +1,127 @@
+local textures = asset.resource({
+  Name = "Cardinal Directions Textures",
+  Type = "HttpSynchronization",
+  Identifier = "cardinal_directions_textures",
+  Version = 1
+})
+
+
+local FadeInConstellationLabels = {
+  Identifier = "os.nightsky.FadeInConstalltionLabels",
+  Name = "Fade In Constelltion Labels",
+  Command = [[openspace.fadeIn("Scene.Constellations.Renderable.Labels",nil)]],
+  Documentation = "Fades in the constllation labels",
+  GuiPath = "/Constellations/Lines",
+  IsLocal = false
+}
+
+local FadeOutConstellationLabels = {
+  Identifier = "os.nightsky.FadeOutConstalltionLabels",
+  Name = "Fade Out Constelltion Labels",
+  Command = [[openspace.fadeOut("Scene.Constellations.Renderable.Labels",nil)]],
+  Documentation = "Fades out the constellation labels",
+  GuiPath = "/Constellations/Lines",
+  IsLocal = false
+}
+
+local ShowConstellationElements = {
+  Identifier = "os.nightsky.ShowConstellationElements",
+  Name = "Show Constelltion Elements",
+  Command = [[
+    local fadeSpeed = openspace.propertyValue("OpenSpaceEngine.FadeDuration")
+    openspace.setPropertyValueSingle("Scene.Constellations.Renderable.Fade", 0)
+    openspace.setPropertyValueSingle("Scene.Constellations.Renderable.Enabled", true)
+    openspace.setPropertyValueSingle("Scene.Constellations.Renderable.Fade", 1, fadeSpeed)
+    openspace.setPropertyValueSingle("Scene.Constellations.Renderable.DrawElements", true)
+  ]],
+  Documentation = "Shows the constllation lines with consideration of label state",
```

Comment:
```suggestion
  Documentation = "Shows the constellation lines with consideration of label state",
```

"with consideration of label state" sounds weird and I don't understand the sentence it enough to suggest an improvement